components/identities/identitiesGalleryV1.tsx

```diff
@@ -64,19 +64,20 @@ const IdentitiesGalleryV1: FunctionComponent<IdentitiesGalleryV1Props> = ({
             onClick={() => router.push(`/identities/${identity.id}`)}
           >
             {needAutoRenewal?.includes(identity.domain) &&
-            isIdentityExpiringSoon(identity) ? (
+             (isIdentityExpired(identity) || isIdentityExpiringSoon(identity)) ? (
               <div className={styles.expiryWarning}>
                 <Tooltip
                   title={
                     isIdentityExpired(identity)
-                      ? "This domain is expired"
-                      : `Be careful, this domain will expire on ${timestampToReadableDate(
+                      ? "This domain has expired! Renew it now to keep using it."
+                      : `This domain will expire on ${timestampToReadableDate(
                           identity?.domain_expiry ?? 0
                         )}`
                   }
                   arrow
                 >
-                  <ErrorIcon color="error" />
+                  {/* <ErrorIcon color="error" /> */}
+                  <RenewalIcon color={isIdentityExpired(identity) ? "#FF3333" : "#FFDE21"} width="12" />
```

Comment:
Why are you changing the icon ? Also the hover tooltip doesn't work for me on your PR

Reply:
Ok, I'll make sure the requested changes are applied, and try the mock address, thanks.

Reply:
`curl https://api.starknet.id/renewal/get_non_subscribed_domains?addr=0x05F1F8De723d8117daA26ec24320d0EacabC53a3D642Acb0880846486e73283a`
gives below result 
["testapple.stark","sdfsdqdsd.stark","testniin.stark","qzdqsddqz.stark","test5.stark","niko.stark","dsqqsdsqds.stark","testnico.stark","testnicooo.stark"]

Reply:
> `curl https://api.starknet.id/renewal/get_non_subscribed_domains?addr=0x05F1F8De723d8117daA26ec24320d0EacabC53a3D642Acb0880846486e73283a` gives below result ["testapple.stark","sdfsdqdsd.stark","testniin.stark","qzdqsddqz.stark","test5.stark","niko.stark","dsqqsdsqds.stark","testnico.stark","testnicooo.stark"]

Yes, because it returns the domains that are not subscribed to auto renewal.

Reply:
Obviously, the ones from response have the red icon from the image you shared. "TESTET.STARK" is missing.

Reply:
After studying [api.starknet.id](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet)/[src](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src)/[endpoints](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src/endpoints)/[renewal](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src/endpoints/renewal)
/get_non_subscribed_domains.rs
The renewal/get_non_subscribed_domains response, filters out some expired domains based on their renewal status. This means even though they are expired, they are not considered "non-subscribed"

Reply:
> > `curl https://api.starknet.id/renewal/get_non_subscribed_domains?addr=0x05F1F8De723d8117daA26ec24320d0EacabC53a3D642Acb0880846486e73283a` gives below result ["testapple.stark","sdfsdqdsd.stark","testniin.stark","qzdqsddqz.stark","test5.stark","niko.stark","dsqqsdsqds.stark","testnico.stark","testnicooo.stark"]
> 
> Yes, because it returns the domains that are not subscribed to auto renewal.

So, can I change this to get all domains, then filter to get expired ones for showing red for `expired` and yellow for `expiring soon`?
Basically, I am asking if I can use another api edpoint apart from `/renewal/get_non_subscribed_domains?` to achieve my task? I am thinking of using `/addr_to_full_ids?addr=` for expired/expiring soon and `/renewal/get_non_subscribed_domains?`  for renew subscription indicator.

Reply:
> Obviously, the ones from response have the red icon from the image you shared. "TESTET.STARK" is missing.

No, e.g.: niko.stark is not expired but is returned by this route as I disabled the autorenewal for it
<img width="902" alt="image" src="https://github.com/user-attachments/assets/03dc46f0-54ec-4a62-9f89-c8388054b0d0" />

Please don't use this route, it has nothing to do with the issue

Reply:
> After studying [api.starknet.id](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet)/[src](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src)/[endpoints](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src/endpoints)/[renewal](https://github.com/lfglabs-dev/api.starknet.id/tree/testnet/src/endpoints/renewal) /get_non_subscribed_domains.rs The renewal/get_non_subscribed_domains response, filters out some expired domains based on their renewal status. This means even though they are expired, they are not considered "non-subscribed"

Yes but that's not what we want

Reply:
Just use the expiry of each domain to determine if it's expired, and then show the red icon to tell the user it's expired

Reply:
Awesome, thanks!